demos/TicTacToe/Classes/TicTacToeLayer.h

```diff
@@ -0,0 +1,65 @@
+#ifndef TICTACTOE_DEMO_CLASSES_TICTACTOELAYER_SCENE_H_
+#define TICTACTOE_DEMO_CLASSES_TICTACTOELAYER_SCENE_H_
+
+#include "TicTacToeScene.h"
+#include "cocos2d.h"
+using cocos2d::Director;
+using cocos2d::Event;
+using cocos2d::Layer;
+using cocos2d::LayerColor;
+using cocos2d::Point;
+using cocos2d::Sprite;
+using cocos2d::Touch;
+
+#include "firebase/app.h"
+#include "firebase/auth.h"
+#include "firebase/database.h"
+
+using firebase::database::DataSnapshot;
+using firebase::database::MutableData;
+using firebase::database::TransactionResult;
+#include <unordered_set>
+
+#include "firebase/future.h"
+#include "firebase/util.h"
+
```

Comment:
nit don't mix includes and using. Put the includes first, and the using statements should follow.